expr/expr.go

```diff
@@ -3613,6 +3613,48 @@ func EvalExpr(e *expr, from, until int32, values map[MetricRequest][]*MetricData
 
 		return results, nil
 
+	case "removeBetweenEpochs": // removeBetweenEpochs(seriesLists, fromEpoch, untilEpoch)
+		args, err := getSeriesArg(e.args[0], from, until, values)
+		if err != nil {
+			return nil, err
+		}
+
+		fromEpoch, err := getIntArg(e, 1)
```

Comment:
Maybe support epochs in whatever format we accept input dates in?  That would require moving `dateParamToEpoch` from the server package to the expr package.  That would also allow the epochs to be `int32` which would remove a bunch of conversion to `int`s.